packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -434,44 +439,72 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
     }, [tileStyle]);
 
     /*
-     * React on changes in the primary color prop.
+     * React on changes in the primary color prop. If not defined in the URL, check if it is defined in app config. Otherwise set to default color.
      */
     useEffect(() => {
-        setPrimaryColor(primaryColor);
-    }, [primaryColor]);
+        const defaultPrimaryColor = '#005655';
+
+        // Sets primary color. Priority: URL/Query Param -> App Config -> Default.
+        if (!isNullOrUndefined(primaryColor)) {
+            setPrimaryColor(primaryColor)
+        } else if (!isNullOrUndefined(appConfig?.appSettings?.primaryColor)) {
+            setPrimaryColor(appConfig?.appSettings?.primaryColor)
+        } else {
+            setPrimaryColor(defaultPrimaryColor);
+        }
```

Comment:
Can this be simplified using a nullish coalescing operator?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing